src/main/kotlin/plugins/scijava/MastodonSidePlugin.kt

```diff
@@ -56,14 +57,16 @@ class MastodonSidePlugin : DynamicCommand() {
     // =============================================================================
     @Plugin(type = Command::class, name = "Mastodon to Sciview: Levels")
     class MastodonSidePluginResLevels : DynamicCommand() {
+        private val logger by lazyLogger()
+
         @Parameter
```

Comment:
hehe, two logging subsystems side by side :-)

maybe we kick out the `LogService` then? it's used only in two cases below anyway...

Reply:
just saying :-)